### DISCUSSION TITLE
Migrate to official docker

### DISCUSSION THREAD

**Discussion by kameleon25** [Ylianst/MeshCentral#5415] [Q&A] [answered]
I have been running a couple of docker instances of meshcentral using the uldiseihenbergs/meshcentral image as it was the only one I could get working years ago when I tried. Now that I see there is an official docker setup, is there a best way to accomplish this without having to redo all the agents? On the one I am running now I have the backup, data, files, and web folders mapped to persistent storage. I cannot find the command I used to start the instances though as that has been a few years now. 

Should it be as simple as editing the docker-compose.yml for the official and point the data locations to the existing files?

**Comment by si458** [comment, marked as answer]
in theory yes
just make sure u map the volumes correctly
`meshcentral-data volume from old image` -> `/opt/meshcentral/meshcentral-data`
`meshcentral-files volume from old image` -> `/opt/meshcentral/meshcentral-files`
`meshcentral-backups volume from old image` -> `/opt/meshcentral/meshcentral-backups`
`meshcentral-webvolume from old image` -> `/opt/meshcentral/meshcentral-web`

https://github.com/Ylianst/MeshCentral/tree/master/docker#docker-composeyml (ignore the env section unless u use it)

**Comment by kameleon25** [comment]
As an update to this: doing as @si458 said worked. However if you are currently running another docker setup double check your config file in the data folder. Mine was still pointing to /home/node/ when this container puts everything in /opt/meshcentral/. So a quick vi search and replace all did the trick.

**Reply by si458** [reply]
glad it worked!
also new hint, the is a new 'master' docker image 😉
which is basically the master branch, if you want all the latest feature before a npm release!

**Reply by kameleon25** [reply]
le sigh..... HA! I'll have to dig into that for sure.
